pkg/snapshot/plugin.go

```diff
@@ -76,6 +77,11 @@ func (p *Plugin) Init(config map[string]string) error {
 		p.pxClient.namespace = defaultNamespace
 	}
 
+	if err := os.Setenv(pxNamespaceKey, p.pxClient.namespace); err != nil {
+		p.Log.Errorf("Failed to set Portworx namespace: %v", err)
+	} else {
+		p.Log.Infof("Using namespace: %v", p.pxClient.namespace)
```

Comment:
Actually what happens if the env var is set?  Px would be in that namespace, but not valero plugin?

Reply:
Isn't the else part true when the env var is set as well?

Reply:
The library essentially looks for the portworx k8s service object in that namespace. Using that k8s service object, it generates a REST URL to talk to. This plugin simply uses the library and gets a VolumeDriver client object.

Reply:
> Isn't the else part true when the env var is set as well?
I am explicitly setting that env variable which is passed in through command line in the plugin
```
velero snapshot-location create portworx-cloud-ns-auth --provider portworx.io/portworx --config type=cloud,credId=06779ef3-aac0-4e36-b652-4412a26fc044,PX_NAMESPACE=portworx
```

Reply:
Ideally the openstorage library should not check for set env variables.

Reply:
make sense. thanks.